app/jobs/pastes_cleanup_job.rb

```diff
@@ -4,7 +4,8 @@
 class PastesCleanupJob < ApplicationJob
   queue_as :default
 
-  def perform(paste)
-    paste.destroy
+  def perform(paste_id)
+    paste = Paste.where(id: paste_id)[0]
```

Comment:
```suggestion
    paste = Paste.find_by(id: paste_id)
```
Not using `find` as that would throw an exception

Reply:
Thanks!